samples/04.0-file-transfer/integration-tests/src/test/java/org/eclipse/dataspaceconnector/extension/sample/test/FileTransferSampleTest.java

```diff
@@ -14,79 +14,38 @@
 import org.eclipse.dataspaceconnector.junit.extensions.EdcRuntimeExtension;
-import org.eclipse.dataspaceconnector.junit.testfixtures.TestUtils;
-import org.eclipse.dataspaceconnector.spi.types.domain.DataAddress;
-import org.eclipse.dataspaceconnector.spi.types.domain.transfer.DataRequest;
-import org.jetbrains.annotations.NotNull;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
-import java.io.File;
-import java.io.IOException;
-import java.time.Duration;
 import java.util.Map;
 
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.awaitility.Awaitility.await;
-import static org.hamcrest.Matchers.emptyString;
-import static org.hamcrest.Matchers.equalTo;
-import static org.hamcrest.Matchers.not;
-
 @EndToEndTest
 public class FileTransferSampleTest {
 
-    static final ObjectMapper MAPPER = new ObjectMapper();
-    static final String INITIATE_CONTRACT_NEGOTIATION_URI = "http://localhost:9192/api/v1/data/contractnegotiations";
-    static final String LOOK_UP_CONTRACT_AGREEMENT_URI = "http://localhost:9192/api/v1/data/contractnegotiations/{id}";
-    static final String INITIATE_TRANSFER_PROCESS_URI = "http://localhost:9192/api/v1/data/transferprocess";
-    static final String CONTRACT_OFFER_FILE_PATH = "samples/04.0-file-transfer/contractoffer.json";
     static final String CONSUMER_CONFIG_PROPERTIES_FILE_PATH = "samples/04.0-file-transfer/consumer/config.properties";
     static final String PROVIDER_CONFIG_PROPERTIES_FILE_PATH = "samples/04.0-file-transfer/provider/config.properties";
-    static final String TRANSFER_FILE_PATH = "samples/04.0-file-transfer/filetransfer.json";
-    // Reuse an already existing file for the test. Could be set to any other existing file in the repository.
-    static final String SAMPLE_ASSET_FILE_PATH = TRANSFER_FILE_PATH;
-    static final String DESTINATION_FILE_PATH = "samples/04.0-file-transfer/consumer/requested.test.txt";
-    static final Duration TIMEOUT = Duration.ofSeconds(15);
-    static final Duration POLL_INTERVAL = Duration.ofMillis(500);
-    static final String API_KEY_HEADER_KEY = "X-Api-Key";
-    static final String API_KEY_HEADER_VALUE = "password";
+    static final String SAMPLE_ASSET_FILE_PATH = "samples/04.0-file-transfer/README.md";
     @RegisterExtension
     static EdcRuntimeExtension provider = new EdcRuntimeExtension(
             ":samples:04.0-file-transfer:provider",
             "provider",
             Map.of(
                     // Override 'edc.samples.04.asset.path' implicitly set via property 'edc.fs.config'.
-                    "edc.samples.04.asset.path", getFileFromRelativePath(SAMPLE_ASSET_FILE_PATH).getAbsolutePath(),
-                    "edc.fs.config", getFileFromRelativePath(PROVIDER_CONFIG_PROPERTIES_FILE_PATH).getAbsolutePath()
+                    "edc.samples.04.asset.path", FileTransferSampleTestUtils.getFileFromRelativePath(SAMPLE_ASSET_FILE_PATH).getAbsolutePath(),
```

Comment:
nit: static import can be used e..g import static org.eclipse.dataspaceconnector.extension.sample.test.FileTransferSampleTestUtils.getFileFromRelativePath;